vrt/config.js

```diff
@@ -0,0 +1,184 @@
+/*
+Copyright (c) 2018-2019 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+/* eslint-disable flowtype/require-valid-file-annotation */
+/* eslint-env node */
+
+module.exports = {
+  'country-select-dropdown': {
+    fullPage: true,
+    interactions: [
```

Comment:
when will we take screenshots? after each command? or before and after?

Reply:
each interaction `behavior` should prepare the page for a screenshot. the actual screenshot is taken **after** the behavior function runs. we kind of capture the before with the default scenario snapshot. if you really need a before/after where there are interactions before each you can add an interaction to the array for both before and after (and name appropriately).

originally i passed in a function so you could take snapshots yourself in the `behavior` function but you can only take one snapshot per test due to jest limitations. i thought maybe with some kind of generator function thing we could do tests dynamically but jest doesn't seem to allow for run time test creation.